.devcontainer/Containerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/go-toolset:latest
+
+USER root
+
+RUN dnf install -y \
+        sudo git-core \
+    && dnf clean all \
+    && rm -rf /var/cache/yum
```

Comment:
git is already installed on these images, make is missing

Reply:
`make` is included, but `git-core` needed a reinstall for some reason, else Codespaces would not find the `git ` binary.

```
$ rpm -qa make
make-4.3-8.el9.x86_64
```

Reply:
It might also be that in the more recent image, like `registry.access.redhat.com/ubi9/go-toolset:latest`, this is now part of this image. This was not the case of ubi8 and also not for slightly earlier released image (this latest is now from 5 days ago).